### PR TITLE
Update labels on burial allowance page

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3836

image

### CODE REVIEW

**Review by cehsu — APPROVED** [review]

File: src/js/burials/labels.jsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ export const locationOfDeathLabels = {
 };
 
 export const allowanceLabels = {
-  nonService: 'Non-service-connected death',
-  service: 'Service-connected death',
+  nonService: 'Non-service-connected death (for Veteran receiving a pension or disability compensation)',
+  service: 'Service-connected death (for Veteran death related to, or a result of, a service-connected disability)',
```
Comment (cehsu) [inline src/js/burials/labels.jsx]: Should this be '...or as a result of'?

**Review by jbalboni — APPROVED** [review]

File: src/js/burials/labels.jsx (hunk 1 above)
Comment (cvalarida) [inline src/js/burials/labels.jsx]: Mmm...this is one of the finer points of grammar I'm not sure about. It _sounds_ like it should be 'as a result of', but I just copied @peggygannon's text from [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3836#issuecomment-314473982).

@peggygannon Should I change it the parenthetical statement to "(for Veteran death related to, or as a result of, a service-connected disability)", or leave it the way it is?

Comment (peggygannon) [inline src/js/burials/labels.jsx]: @cvalarida Hi, thanks, for checking. We don't need to add "as" here, but we should change "a" to "the".  I've updated to this: 
for Veteran death related to, or the result of, a service-connected disability.

Comment (cvalarida) [inline src/js/burials/labels.jsx]: Awesome. Thanks!

**Comment by peggygannon** [comment]
@jbalboni sorry to do this to you, but I can I make two more edits to the Burial Allowance parenthetical copy in time for the public launch next week? 

It should read: (for a Veteran death related to, or resulting from, a service-connected disability.

**Pull request merged**